packages/eslint-config/react.js

```diff
@@ -1,90 +1,124 @@
-const core = require("./core");
+import coreConfig from "./core.js";
+import reactIntlFormatPlugin from "@calm/eslint-plugin-react-intl";
```

Comment:
Does this still work? It seems to me that you removed a lot of valid disables for this rule.

Reply:
hopefully it's still working :-D.

...

It's still working, as far as I can see. I am not aware that I have removed rules concerning @calm/eslint-plugin-react-intl. Which one do you exactly mean?

Reply:
For instance, here: https://github.com/vivid-planet/comet/pull/3230/files#diff-8d6a721eda83ff6ed30717ed6d4ad3b4eceaa355208edf8214e4f3181eb7af1dL1

This should cause lint errors with the removed ESLint disable.

Reply:
no, that's totally fine. 

the rule `/* eslint-disable @calm/react-intl/missing-formatted-message */` got fully disabled in storybook - see https://github.com/vivid-planet/comet/pull/3230/files#diff-adb50cd1dffceb575084b96e970a089423175fb84e7a4d4a04c50a7484499ce2R13

Eslint complained now that we are ignoring a rule which did not even trigger, thats why i removed it.

Reply:
The file I linked isn't part of Storybook, it's a unit test in the admin package.

Reply:
indeed, the same applies here: https://github.com/vivid-planet/comet/pull/3230/files#diff-7a156624eceabfbd2af06f0fa6b45882b184cd16778e70c8fdbc165e80fb993aR20

All `@calm/react-intl/missing-formatted-message` are deactivated in tests.